script.js

```diff
@@ -1,4 +1,425 @@
+var twoPlayer_LOWESTannounceWinner = `announce winner for Lowest Combined Number game before returning to P1`;
+var varNumofDice = `mode with variable num of dice`;
+var varNumofDice_rollDice = `roll dice for variable num of dice mode`
+var varNumofDice_announceWinner = `var num of dice - winner`
+// Define initial Game Mode!
+var gameMode = startGame;
+
+// Define global variables
+var diceRolls = [];
+var currPlayer = 1;
+var nextPlayer = 1;
+
+var playerGuess = [];
+var P1Guesses = [];
+var P2Guesses = [];
+
+// Global Variables for Multi-player game
+var numPlayers = 2;
+var numRoundsPlayedByEachPlayer = Array(numPlayers).fill(0);
+var numWinsOfEachPlayer = Array(numPlayers).fill(0);
+
+//function to select game mode
+var inputGameModes = function (input) {
+  var outputMessage = ``;
+  if (input.toUpperCase() == "NORMAL") {
+    gameMode = twoPlayer_EnterDice;
+    outputMessage = `Hit submit to start the Normal 2 player BEAT THAT game!`;
+  } else if (input.toUpperCase() == "LOWEST") {
+    gameMode = twoPlayerLowest_EnterDice;
+    outputMessage = `Hit submit to start the lowest combined number - 2 player BEAT THAT game!`;
+  } else if (input.toUpperCase() == "AUTO") {
+    gameMode = auto_twoPlayer_EnterDice;
+    outputMessage = `Hit submit to start the Auto generated combined number - 2 player BEAT THAT game!`;
+  } else if (input.toUpperCase() == 'VAR DICE'){
+  gameMode = varNumofDice;
+  outputMessage = `Hit submit to start the variable number of dice (2 player) BEAT THAT game!`;
+  } else {
+    //Input validation
+    outputMessage = `Input error. <br> Please select your game modes as follows: <br> 
+    Input 'NORMAL' for normal 2 player mode<br>
+    Input 'LOWEST' for lowest combined number - 2 player mode<br>
+    Input 'VAR DICE' for variable number of dice mode`;
+  }
+  return outputMessage;
+};
+
+//function to track total running score of a player
+var generateSumOfGuesses = function (ArrayOfPLayerGuesses) {
+  var sumOfGuesses = 0;
+  for (var i = 0; i < ArrayOfPLayerGuesses.length; i += 1) {
+    sumOfGuesses = sumOfGuesses + Number(ArrayOfPLayerGuesses[i]);
+  }
+  return sumOfGuesses;
+};
+//========== Function for 2 Player Normal mode ==========
+var startTwoPlayerNormalMode = function (input) {
+  // 2 PLAYER NORMAL GAME - P1 START
+  if (nextPlayer == 1 && gameMode == twoPlayer_EnterDice) {
+    diceRolls = [];
+    currPlayer = 1;
+    //number of dice rolls is 2 by default
+    for (var counter = 0; counter < 2; counter += 1) {
+      diceRolls.push(rollDice());
+    }
+    console.log(`Dice rolls: ${diceRolls}`);
+
+    if (diceRolls[0] > diceRolls[1]) {
+      playerGuess = diceRolls[0] + "" + diceRolls[1];
+    } else if (diceRolls[0] == diceRolls[1]) {
+      playerGuess = diceRolls[0] + "" + diceRolls[1];
+    } else if (diceRolls[0] < diceRolls[1]) {
+      playerGuess = diceRolls[1] + "" + diceRolls[0];
+    }
```

Comment:
Can try refactoring lines 87-93 into a helper function since you use it multiple times (eg. in 116-122).